dotNETStandard/HealthVaultProviderManagementPortal/HealthVaultProviderManagementPortal/Controllers/PatientController.cs

```diff
@@ -113,6 +113,11 @@ public class PatientController : Controller
                 queryParameters.Add($"endDate={endDate.Value.ToString("yyyy-MM-dd", CultureInfo.InvariantCulture)}");
             }
 
+            if (timeZone != null)
```

Comment:
Since timeZone is required for the API endpoint, if timeZone was null wouldn't we want to error rather than attempt the call?